isolate_example/android/app/build.gradle

```diff
@@ -22,23 +22,28 @@ if (flutterVersionName == null) {
 }
 
 apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
 apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
 android {
     compileSdkVersion 28
 
+    sourceSets {
+        main.java.srcDirs += 'src/main/kotlin'
+    }
+
     lintOptions {
         disable 'InvalidPackage'
     }
 
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
-        applicationId "dev.flutter.isolate_example"
+        applicationId "com.example.isolate_example"
```

Comment:
Is this a necessary change? It's not hugely important, but having all our samples aligned on their domains makes my inner Marie Kondo happy.

Reply:
It's what the scripts now generate. To change it back requires some fairly tedious work, moving files around and manually editing configuration files, that will discourage keeping the samples up to date. I understand the desire, but realistically we should just go with whatever the `flutter create` script creates.

Reply:
Using the `--org` parameter should take any manual work out of it:

`flutter create --org=dev.flutter project_name`

If that doesn't work, though, and you'd be required to manually edit the files, then definitely ignore me here.

Reply:
Ooh nice